README.md

```diff
@@ -224,6 +224,54 @@ amadeus.shopping.flightOffers.get({
     console.log(responseError);
 });
 
+// Flight Create Orders
+amadeus.shopping.flightOffersSearch.get({
```

Comment:
@akshitsingla I think in the README we don't need to have such an explicit example. For example in Python's README we have:
`# Flight Create Orders
amadeus.booking.flight_orders.post(flights[0], traveler)`

We do have a separate real [examples](https://github.com/amadeus4dev/amadeus-python/tree/master/examples) and the one for Flight Create Orders is in this [PR](https://github.com/amadeus4dev/amadeus-python/pull/68)

Reply:
Fixed